### PR TITLE
[JENKINS-70695] add copy to clipboard button for snippet generator

### PR DESCRIPTION
**Pull request by mawinter69**
this change adds a copy to clipboard button below the field with the generated code

See [JENKINS-70695](https://issues.jenkins.io/browse/JENKINS-70695)

- [x] Make sure you are opening from a **topic/feature/bugfix branch** (right side) and not your main branch!
- [x] Ensure that the pull request title represents the desired changelog entry
- [x] Please describe what you did
- [x] Link to relevant issues in GitHub or Jira
- [ ] Link to relevant pull requests, esp. upstream and downstream changes
- [ ] Ensure you have provided tests - that demonstrates feature works or fixes the issue

### CODE REVIEW

**Review by raul-arabaolaza — APPROVED** [review]

**Comment by mawinter69** [comment]
affects #599

**Review by jglick — COMMENTED** [review]
Looks reasonable though IIUC it is tricky to test locally since only `https`-protocol pages will actually let you copy. https://github.com/jglick/jenkins-demo-reverse-proxy might help.

**Comment by jglick** [comment]
> testing with `mvn hpi:run` works fine

Ah TIL.

File: plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js [hunk 1]
```diff
@@ -21,6 +21,9 @@ function handlePrototype(url, crumb) {
             if (response.ok) {
                 response.text().then((responseText) => {
                     document.getElementById('prototypeText').value = responseText;
+                    copybutton = document.querySelector('.jenkins-copy-button');
```
[the remark this answers is not in the document]
Comment (jglick) [reply plugin/src/main/resources/org/jenkinsci/plugins/workflow/cps/Snippetizer/handle-prototype.js:24]: Probably not relevant for production uses.

**Review by jglick — APPROVED** [review]
Ah I normally run with the `wildcardDNS` Maven property set which I guess deactivates the copy button. With that profile disabled I am able to test locally.

**Pull request merged**